transcendental/load-libs.lisp

```diff
@@ -5,8 +5,17 @@
 (in-package #:magicl-transcendental.foreign-libraries)
 
 (cffi:define-foreign-library libexpokit
-  (:darwin "expokit.dylib")
-  (:unix  "expokit.so")
+  (:darwin (:or #.(merge-pathnames "libexpokit.dylib"
+                                   (or *compile-file-truename*
+                                       *load-truename*))
+                "libexpokit.dylib"
+                "expokit.dylib"))
+  (:unix  (:or #.(merge-pathnames "libexpokit.so"
+                                  (or *compile-file-truename*
+                                      *load-truename*))
+               "libexpokit.so"
+               "expokit.so"))
+
   (t (:default "expokit")))
 
 (push 'libexpkit magicl.foreign-libraries::*cffi-libraries*)
```

Comment:
Should that be `'libexpokit`?

Reply:
Hmm, I don't know about that bit - it's from @tarballs-are-good. Since it's a typo and nothing broke I guess it can be removed entirely?

Reply:
Nope, shouldn't be removed - correcting it gives nice output in `print-availability-report`. Will fix.

Reply:
good find